mmcv/ops/csrc/pytorch/focal_loss.cpp

```diff
@@ -65,6 +79,15 @@ void sigmoid_focal_loss_forward(Tensor input, Tensor target, Tensor weight,
                                     alpha);
 #else
     AT_ERROR("SigmoidFocalLoss is not compiled with GPU support");
+#endif
+  } else if (input.device().type() == at::kMLU ){
```

Comment:
The build failed on host without mlu device.
Better wrap the condition inside `#ifdef MMCV_WITH_MLU`

Reply:
Yes, I moved this line inside the macro define